changelog.txt

```diff
@@ -1,4 +1,4 @@
-
+ 
```

Comment:
Do you mind updating changelog properly? Otherwise we need to manually construct it every time when release is done and it is very slow process.

Reply:
Added, but not sure if that is the right version number and the correct message. Would you please double-check?